README.md

```diff
@@ -40,12 +40,12 @@ and even better, on a video projector in your local hackspace!
 
 ## How it works
 
-Images are crawled from multiple public pre-configured sources.  
```

Comment:
could you revert the removal of the trailing spaces.
they were intended for a line break